client/components/GamePage.js

```diff
@@ -94,7 +106,10 @@ export default () => {
         <Grid container spacing={3} className={classes.wrapper}>
           <Grid item xs={12} sm={2} className={classes.players}>
             <UserList />
-            <Info />
+            <Grid item xs={12} sm={2} className={classes.bottomButtons}>
```

Comment:
Do we need a Grid element here or can this be a simple `<div>` instead? I think some of the css rules can also be cleaned up, especially `overflowY` which causes issues with the Info button hover.

Reply:
Yeah, I didn't notice the info button hover was broken. Should be fixed

desktop: <img width="1193"  src="https://user-images.githubusercontent.com/570079/99201025-d2220080-2776-11eb-8568-b908a71103ac.png">

mobile: <img width="258"  src="https://user-images.githubusercontent.com/570079/99201029-d51cf100-2776-11eb-9ed7-d75918600aee.png">